docs/release-notes.md

```diff
@@ -1,45 +1,41 @@
 
-#### [RPC](Glossary.md#r)
-- **Docker support added:** Added a dockerfile to RPC that allows customers to build and execute RPC in a docker container on the Intel&reg; AMT device.
+    ```html
+    https://example.url/mps
+    https://example.url/rps
+    ```
+    
+    With the addition of Kong and its rich plug-in support, this opens up options for integrators to utilize a broader set of user authentication services.  For example, the same tokens that are already being used by the rest of a device management tool to authenticate users can also be used to authenticate users with the Open AMT Cloud Toolkit service(s).  This new feature should make integration of Open AMT Cloud Toolkit microservices into existing management consoles much more seamless.
 
+### Modifications and Removals
+#### RPS
+- **RPS Activation Changes:** In 1.2 we added a short delay between the time we activated an AMT device and when we configured the AMT features.  While RPS waiting for AMT to be ready, RPS and RPC would send "heartbeats" back and forth to keep the connection alive.  This "heartbeat" rate was very fast and would cause thousands of progress dots to be output to the client terminal screen while this was happening.  In 1.3 we have fixed this "heartbeat" pace to only 1 "heartbeat" every 5 seconds.  Many dots (not to mention packets) were saved with this change.
+#### UI Toolkit
+- **Deprecation announcement of non-socket UI Toolkit components:** Following the deprecation announcement in 1.2, we have removed the following UI Toolkit components: Profile Editor, CIRA Editor, Domain Editor, Audit Log, and Device List.  KVM and SOL UI Toolkit modules will remain in the UI Toolkit.  Look for broader UI framework support coming in future releases.
+#### MPS & RPS
+- **Removal of Developer Mode:** As was announced in 1.2 release, we removed "Developer Mode".  In "Developer Mode" our services utilized a file-based approach to storing data.  Operating in this mode had several potentially serious issues from security of the data stored in the files, scaling performance, not to mention the additional code required to support reading from a file versus accessing an API.  With this change, the MPS and RPS will exclusively utilize the DB and Vault interfaces for storing and accessing data.  The recommended deployment model going forward is to utilize containers (docker).
+- **API refactoring is complete:** In 1.2 we refactored the RPS REST APIs and in 1.3 we performed a similar refactoring the MPS REST APIs.  With the release of 1.3, these API breaking changes have been completed.  Special note: with the addition of Stateless Authenication, the API key in the header has been removed.  Please take a look at our updated Swagger documentation: 
```

Comment:
What is the big take-away with the refactoring? Is it that the code became simpler, more legible?

Reply:
good point, I'll add a take away sentence in here as well.